### PR TITLE
Fraunhofer FKIE DDoS attack feed parser bot

### PR DESCRIPTION
**Pull request by sascha-jopen**
Fraunhofer FKIE is tracking DDoS botnets and provides APIs for reporting detected C&C servers as well as detected DDoS attack events. This parser, together with HTTP (stream) collectors, allows for feeding these events into IntelMQ.
Please review and make suggestions for including that code back into the main IntelMQ repository.

### CODE REVIEW

File: intelmq/bots/parsers/fraunhofer/parser_ddosattack.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        event.add('source.port', message['cnc']['port'])
+
+        if message['messagetype'] != 'cnc_message' and hasattr(self.parameters, 'unknown_messagetype_accuracy'):
+            event.add('feed.accuracy', self.parameters.unknown_messagetype_accuracy, overwrite=True)
+
+        yield event
+
+    def __parse_ddos_targets(self, message, line, report):
+        if message['messagetype'] != 'cnc_message':
+            self.logger.info('Unable to create ddos events due to '
```
[the remark this answers is not in the document]
Comment (sascha-jopen) [reply intelmq/bots/parsers/fraunhofer/parser_ddosattack.py]: Messages from our feed contain information about a C&C server and one or multiple DDoS attack targets in a single message. We can easily raise an exception, if the C&C type is unknown to us. In that case we wouldn't emit any event, anyway. However, if we know the C&C type, but not the attack type, we still want to emit the C&C event, which is still valid and valuable. Raising an exception also suppresses that event. I'm going to split the parser into two parsers then. One for extracting the cnc events and one for the ddos events. This also makes it more flexible to put things together. Or do you have a better idea to deal with that problem?

**Comment by sascha-jopen** [comment]
I just added a new commit. As requested I updated the feeds.yaml file. Furthermore, I splitted the two parser components into two distinct parsers to support raising ValueErrors in case of unknown message formats. The documentation and tests were updated accordingly. Any additional comments or requests?

**Comment by sascha-jopen** [comment]
Sorry, I completely missed your comment. I added a contact address to the feed information. We do not have a webpage for the feed, nor any other publicly available information regarding our feeds.

**Comment by wagner-certat** [comment]
Thanks! And Merged.

Follow-up commit: 9eed9824e

**Pull request merged**